portfolio_tool_scripts/populate_holdings.gs

```diff
@@ -56,7 +56,7 @@ function get_latest_dividends_data() {
 
   var header_list = ws.getRange(1,1, 1, ws.getRange("A1").getDataRegion().getLastColumn()).getValues()[0];
 
-  var ticker_row_list = ws.getRange(2,1, ws.getRange("A2").getDataRegion().getLastRow(), 12).getValues();
```

Comment:
Haha, great catch on this. It think I was debugging it when I was first developing it and didn't realize that I had hard coded that value.